logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/creating/iterable/contains/creators/impl/InAnyOrderOnlyAssertionCreator.kt

```diff
@@ -53,12 +53,11 @@ abstract class InAnyOrderOnlyAssertionCreator<E, T : IterableLike, in SC>(
                     createExplanatoryGroupForMismatchesEtc(list, WARNING_ADDITIONAL_ELEMENTS)
                 })
             }
-            assertions.add(
+            val sizeFeatureAssertion =
```

Comment:
Please use the `size` feature extractor from CollectingLike -> see ch.tutteli.atrium.logic.creating.maplike.contains.creators.impl.DefaultMapLikeContainsAssertions => line 194

Note that the additional entries should stay inside of `contains...` in reporting and no longer be under `size`
in the end we want to see something like:
```
expect: [1, 2]        (java.util.Arrays.ArrayList <410495873>)
◆ size: 2        (kotlin.Int <710239027>)
  ◾ to be: 1        (kotlin.Int <110771485>)
◆ contains only, in order: 
  ✔ ▶ entry 0: 1        (kotlin.Int <110771485>)
      ◾ to be: 1        (kotlin.Int <110771485>)
    ❗❗ additional entries detected: 
       ⚬ entry 1: 2        (kotlin.Int <710239027>)
```

Reply:
I got this result, but when encountering additional entries usually I get only size assertion printed out, unless I add size feature assertion like the old style, with only mismatches it works as expected

Reply:
I leave a review comment. You need to wrap size and contains assertion in an invisible assertion group. This way they are evaluated together.